Storage was changed from TinyDB to Mongo.

v1-wizard opened the pull request.

Commit:
Storage was changed from TinyDB to Mongo.

Gr1N commented on Makefile:
```diff
@@ -0,0 +1,14 @@
+	@echo "  prepare        => to execute venv deps"
+
+venv:
+	@virtualenv -p python3 venv
```
в свежих питонах можно не использовать отдельный пакет `virtualenv` https://docs.python.org/3/library/venv.html#creating-virtual-environments

v1-wizard replied:
Исправлено. Спасибо за инфу.

Gr1N commented on docker-compose.yml:
```diff
@@ -0,0 +1,20 @@
+version: "3"
+services:
+  app:
```
твой сервис зависит от могли, стоит использовать https://docs.docker.com/compose/compose-file/#depends_on

v1-wizard replied:
Исправлено.

Gr1N commented on requirements.txt:
```diff
@@ -1,3 +1,3 @@
 cerberus==1.1
-tinydb==3.7.0
\ No newline at end of file
+pymongo==3.6.1
```
по хорошему надо использовать асинхронный драйвер к монге https://github.com/mongodb/motor/

v1-wizard replied:
Теперь используется. Спасибо.

Gr1N reviewed (changes requested):
changes requested

Commit:
Mongo driver was changed to Motor!
Fix in docker-compose.yml and MakeFile after review.
Default app port was changed to 8888.

Commit:
Fix import.
